src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -988,11 +988,11 @@ public Response assignGameBoards(@Context final HttpServletRequest request,
         try {
             RegisteredUserDTO currentlyLoggedInUser = userManager.getCurrentRegisteredUser(request);
 
-            // Assert user is allowed to set assignments
-            boolean userIsTeacherOrAbove = isUserTeacherOrAbove(userManager, currentlyLoggedInUser);
+            // Assert user is allowed to set assignments - tutors and above are allowed to do so
+            boolean userIsTutorOrAbove = isUserTutorOrAbove(userManager, currentlyLoggedInUser);
             boolean userIsStaff = isUserStaff(userManager, currentlyLoggedInUser);
-            if (!userIsTeacherOrAbove) {
-                return new SegueErrorResponse(Status.FORBIDDEN, "You need a teacher account to create groups and set assignments!").toResponse();
+            if (!userIsTutorOrAbove) {
+                return new SegueErrorResponse(Status.FORBIDDEN, "You need at least a tutor account to create groups and set assignments!").toResponse();
```

Comment:
Maybe nobody sees this message, but "You need a tutor or teacher account ..." is better; staff don't need this message, and users may not know what "at least" means here?